### PR TITLE
[ECS] enable NPM in the agent

### PR DESCRIPTION
**Pull request by nplanel**
What does this PR do?
---------------------

Enable NPM in the agent for ECS deployment

Which scenarios this will impact?
-------------------

Motivation
----------

Additional Notes
----------------

### CODE REVIEW

File: components/datadog/agent/ecs.go [hunk 1]
```diff
@@ -116,6 +116,10 @@ func ecsLinuxAgentSingleContainerDefinition(e config.CommonEnvironment, apiKeySS
 				Name:  pulumi.StringPtr("DD_PROCESS_AGENT_ENABLED"),
 				Value: pulumi.StringPtr("true"),
 			},
+			ecs.TaskDefinitionKeyValuePairArgs{
+				Name:  pulumi.StringPtr("DD_SYSTEM_PROBE_NETWORK_ENABLED"),
```
Comment (pducolin) [inline components/datadog/agent/ecs.go:120]: **❓ question**
Should this be an option ? Not sure if we want to enable it by default

Comment (nplanel) [reply components/datadog/agent/ecs.go:120]: At the moment we have no option passing through for ECS, but I agreed would be better to have the same options than others environments.

Comment (nplanel) [reply components/datadog/agent/ecs.go]: I added ecsagentparams setting (for example) environment variables to the agent

**Review by pducolin — APPROVED** [review]
Thank you !

**Pull request merged**